charts/gateway/templates/tests/test-connection.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   initContainers:
     - name: extract-persistence-sql
       image: "trinodb/trino-gateway"
+      imagePullPolicy: IfNotPresent
```

Comment:
`IfNotPresent` is the default value for `imagePullPolicy`, we don't need to specify it here.